cmd/kubeadm/app/preflight/checks.go

```diff
@@ -279,7 +279,7 @@ func (fac FileAvailableCheck) Name() string {
 
 // Check validates if the given file does not already exist.
 func (fac FileAvailableCheck) Check() (warnings, errorList []error) {
-	klog.V(1).Infof("validating the existence of file %s", fac.Path)
+	klog.V(1).Infof("validating the existence of file %s, expect not exist", fac.Path)
```

Comment:
i think the error message is fine here.
Infof also has one %s and we feed fac.Path to that.

Reply:
Last several days, when I was bootstrapping a new Kubernetes cluster with `kubeadm`, `kubeadm` stucked and service `kubelet` keep failed with "/var/lib/kubelet/config.yaml" no such file and diectoy.

Then I used `kubeadm join -v=5` again, it told me "validating the existence of file /var/lib/kubelet/config.yaml".

That really confused me, because I do not know kubeadm expects that file to exist or not. That log brings more confusion when profiling the failure with `kubeadm join`. 

> Until I read the codes, I finally found that `kubeadm` wants to keep that file "Available" which means not exist.  So I make this PR.

Reply:
Kubeadm creates the file, but later if it already exists it will throw an error.

Why are getting these errors?

I have not seen other reports from users being confused about the message:

> 
"validating the existence of file /var/lib/kubelet/config.yaml".

This is the same as "checking if file already exists", so maybe we can change it to that.

@SataQiu WDYT?

Reply:
> Kubeadm creates the file, but if it already exists on preflight it will throw an error.

Yes, it would. ❤️

> Why are you getting these errors?

That is not an error message. kubeadm works well,  that "V(1).Info"/ "DEBUG" level message just makes me confusing when I profiling with kubeadm.

Again, kubeadm works always well, this PR is not relates to any error.

BTW,  the root cause of the problem occurring on me is about the networking and etcd cluster, it's not related to these files. :) The debug message just confuse me, so I try to make this PR for a better description.

Reply:
🤣🤣🤣 It's up to you to decide whether appending these descriptions are meaningful or not.

Reply:
Yeah, lets keep it the way it is for now.
But if more users complain we can change it.

Thanks for the PR.
/close